tests/test_github_requirements.py

```diff
@@ -115,29 +111,20 @@ def _test_require_approval_username(self, project, job):
         self.assertEqual(len(self.history), 0)
 
         # Add an approval (review) from derp
-        A.addReview('derp', 'APPROVE')
```

Comment:
So this only alters the query side of it, testing the PR review webhook and corresponding trigger happens elsewhere (see test_github.py:test_review_event)